Index: factorial.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/factorial.py b/factorial.py
new file mode 100644
--- /dev/null	(date 1662644301317)
+++ b/factorial.py	(date 1662644301317)
@@ -0,0 +1,9 @@
+number = int(input("Type a number: "))
+factorial = number
+if number <= 0:
+    print("Thanks and bye!")
+else:
+    for i in range(1, number):
+        factorial = factorial * i
+        print(f"i is now {i} so Factorial {factorial}")
+    print(f"The factorial of {number} is {factorial}")
\ No newline at end of file
